src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/properties/StringProperty.scala

```diff
@@ -83,6 +83,15 @@ object StringProperty {
   case class Pattern(value: String) extends JsonSchemaProperty with StringProperty {
     val keyword: Keyword = Keyword.Pattern
   }
+
+  /**
+    * Class representing `$schema` keyword
+    *
+    * @see https://json-schema.org/understanding-json-schema/reference/schema.html
+    */
+  case class SchemaUri(value: String) extends JsonSchemaProperty with StringProperty {
```

Comment:
I think value should be `URI` https://tools.ietf.org/html/draft-zyp-json-schema-04#section-6, not every string isi valid `$schema`